networking/network_observability/network-observability-overview.adoc

```diff
@@ -32,23 +32,28 @@ The Network Observability Operator provides the Flow Collector API custom resour
 {product-title} console integration offers overview, topology view and traffic flow tables.
 
 [id="network-observability-dashboards"]
-=== Network Observability metrics
+=== Network Observability metrics dashboards
 
-The {product-title} console offers the *Overview* tab which displays the overall aggregated metrics of the network traffic flow on the cluster. The information can be displayed by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics.
+On the *Overview* tab in the {product-title} console, you can view the overall aggregated metrics of the network traffic flow on the cluster. You can choose to display the information by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics. *Infrastructure* and *Application* metrics are shown in a split-view for namespace, pod, and workloads. 
```

Comment:
```suggestion
On the *Overview* tab in the {product-title} console, you can view the overall aggregated metrics of the network traffic flow on the cluster. You can choose to display the information by node, namespace, owner, pod, and service. Filters and display options can further refine the metrics.
```

Infra / App splitting doesn't apply to Network Traffic page and its tabs. It's only part of "Dashboards" page.

Reply:
Ah, got it.

Reply:
I removed this and relocated it to the two areas you suggested for the Dashboards but for some reason, the next two don't show as "outdated". I verified the changes are in the preview.